### PR TITLE
[#872][FOLLOWUP] feat(tez): Modify utils and add test case

### PR DESCRIPTION
**Pull request by lifeSo**
### What changes were proposed in this pull request?

modify utils and add test case

### Why are the changes needed?

Fix: #872 

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?
unit test

### CODE REVIEW

File: client-tez/src/test/java/org/apache/tez/common/RssTezUtilsTest.java [hunk 1]
```diff
@@ -158,4 +163,25 @@ public void attemptTaskIdTest() {
     TezTaskAttemptID tezTaskAttemptID = TezTaskAttemptID.fromString(uniqueIdentifierToAttemptId);
     assertEquals(originalTezTaskAttemptID, tezTaskAttemptID);
   }
+
+  @Test
+  public void testParseRssWorker() {
+    Map<Integer, Set<ShuffleServerInfo>> rssWorker = new HashMap<>();
+    int shuffleId = 1001602;
+    String hostnameInfo = "localhost;1001602=172.19.193.152:19999+0_1_2_3,172.19.193.153:19999+2_3_4_5";
```
Comment (jerqi) [inline client-tez/src/test/java/org/apache/tez/common/RssTezUtilsTest.java:171]: What's meaning of `2_3_4_5`? Could we add some assertion for it?

File: client-tez/src/main/java/org/apache/tez/common/IdUtils.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+import org.apache.hadoop.yarn.api.ApplicationConstants;
+import org.apache.hadoop.yarn.api.records.ApplicationAttemptId;
+import org.apache.hadoop.yarn.api.records.ContainerId;
+import org.apache.tez.dag.records.TezTaskAttemptID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class IdUtils {
+  private static final Logger LOG = LoggerFactory.getLogger(RssTezUtils.class);
```
Comment (jerqi) [inline client-tez/src/main/java/org/apache/tez/common/IdUtils.java:28]: RssTezUtils -> IdUtils.

Comment (lifeSo) [reply client-tez/src/main/java/org/apache/tez/common/IdUtils.java:28]: ？ 

yes ,som utils moved from RssTezUtils to IdUtils.

File: client-tez/src/test/java/org/apache/tez/common/RssTezUtilsTest.java (hunk 1 above)
Comment (lifeSo) [reply client-tez/src/test/java/org/apache/tez/common/RssTezUtilsTest.java:172]: 0_1_2_3 is consist of partition id.

and we add comment and

File: client-tez/src/main/java/org/apache/tez/common/IdUtils.java (hunk 2 above)
Comment (lifeSo) [reply client-tez/src/main/java/org/apache/tez/common/IdUtils.java]: we modified it

File: client-tez/src/test/java/org/apache/tez/common/RssTezUtilsTest.java (hunk 1 above)
Comment (jerqi) [reply client-tez/src/test/java/org/apache/tez/common/RssTezUtilsTest.java:172]: Could we add some test cases to verify the correctness of partition id?

**Comment by jerqi** [comment]
> image
> 
> why hadoop test fail ?

Flaky test.

**Comment by lifeSo** [comment]
> > image
> > why hadoop test fail ?
> 
> Flaky test.


should we modify something to make all test ok ? 
Or we can ignore the test fail

**Comment by jerqi** [comment]
> > > image
> > > why hadoop test fail ?
> > 
> > 
> > Flaky test.
> 
> should we modify something to make all test ok ? Or we can ignore the test fail

It's hard to fix flaky test. They are unstable. We can ignore first.

**Comment by lifeSo** [comment]
Then it effect my MR, how to do ?

File: client-tez/src/test/java/org/apache/tez/common/RssTezUtilsTest.java (hunk 1 above)
Comment (lifeSo) [reply client-tez/src/test/java/org/apache/tez/common/RssTezUtilsTest.java:173]: done

**Comment by jerqi** [comment]
> Then it effect my MR, how to do ?

Committer can rerun flaky test.

**Review by jerqi — APPROVED** [review]
LGTM, thanks @zhengchenyu @lifeSo

**Pull request merged**